src/app/common/stores/applications/SsvWeb/Validator.store.ts

```diff
@@ -18,22 +18,14 @@ import ClusterStore from '~app/common/stores/applications/SsvWeb/Cluster.store';
 import AccountStore from '~app/common/stores/applications/SsvWeb/Account.store';
 import MyAccountStore from '~app/common/stores/applications/SsvWeb/MyAccount.store';
 import ApplicationStore from '~app/common/stores/applications/SsvWeb/Application.store';
-// import { getFixedGasLimit, getRegisterValidatorGasLimit } from '~lib/utils/gasLimitHelper';
 import NotificationsStore from '~app/common/stores/applications/SsvWeb/Notifications.store';
 import OperatorStore, { IOperator } from '~app/common/stores/applications/SsvWeb/Operator.store';
 import ProcessStore, { SingleCluster } from '~app/common/stores/applications/SsvWeb/Process.store';
 import { RegisterValidator } from '~app/common/stores/applications/SsvWeb/processes/RegisterValidator';
+import { KeySharesItem } from 'ssv-keys';
```

Comment:
Move third-party package imports (not from project itself) to the top level

Reply:
done